tee-worker/ts-tests/common/utils/identity-helper.ts

```diff
@@ -47,7 +46,7 @@ export async function buildIdentityHelper(
 export async function buildIdentityTxs(
     context: IntegrationTestContext,
     signers: KeyringPair[] | KeyringPair,
-    identities: LitentryIdentity[],
+    identities: any[],
```

Comment:
Can we use a less promiscuous type here?

Reply:
fixed!